tests/appsec/test_automated_login_events.py

```diff
@@ -1347,3 +1223,665 @@ def validate_iden(meta):
+
+            # mandatory
+            assert meta["appsec.events.users.login.failure.usr.login"] == "sdkUser"
+            assert meta["_dd.appsec.usr.login"] == INVALID_USER_HASH
+            assert meta["_dd.appsec.events.users.login.failure.auto.mode"] == "anonymization"
+            assert meta["appsec.events.users.login.failure.track"] == "true"
+            assert meta["_dd.appsec.events.users.login.failure.sdk"] == "true"
+            assert meta["appsec.events.users.login.failure.usr.exists"] == "true"
+
+    def setup_signup_local(self):
+        self.r_success = weblog.post("/signup", data=login_data(context, NEW_USER, PASSWORD))
+
+    def test_signup_local(self):
+        assert self.r_success.status_code == 200
+        for _, trace, span in interfaces.library.get_spans(request=self.r_success):
+            assert_priority(span, trace)
+            meta = span.get("meta", {})
+
+            # mandatory
+            assert meta["appsec.events.users.signup.usr.login"] == NEW_USERNAME_HASH
+            assert meta["_dd.appsec.usr.login"] == NEW_USERNAME_HASH
+            assert meta["_dd.appsec.events.users.signup.auto.mode"] == "anonymization"
+            assert meta["appsec.events.users.signup.track"] == "true"
+
+            # optional (to review for each library)
+            assert meta["appsec.events.users.signup.usr.id"] == NEW_USER_HASH
+            assert meta["_dd.appsec.usr.id"] == NEW_USER_HASH
+
+
+DISABLED = ("datadog/2/ASM_FEATURES/auto-user-instrum/config", {"auto_user_instrum": {"mode": "disabled"}})
+IDENTIFICATION = ("datadog/2/ASM_FEATURES/auto-user-instrum/config", {"auto_user_instrum": {"mode": "identification"}})
+ANONYMIZATION = ("datadog/2/ASM_FEATURES/auto-user-instrum/config", {"auto_user_instrum": {"mode": "anonymization"}})
+
+
+@rfc("https://docs.google.com/document/d/1RT38U6dTTcB-8muiYV4-aVDCsT_XrliyakjtAPyjUpw")
+@features.user_monitoring
+@scenarios.appsec_auto_events_rc
+class Test_V3_Login_Events_RC:
+    def _send_rc_and_execute_request(self, config):
+        config_state = rc.rc_state.set_config(config).apply()
+        request = weblog.post("/login?auth=local", data=login_data(context, USER, PASSWORD))
+        return {"config_state": config_state, "request": request}
+
+    def _assert_response(self, test, validation):
+        config_state, request = test["config_state"], test["request"]
+
+        assert config_state[rc.RC_STATE] == rc.ApplyState.ACKNOWLEDGED
+        assert request.status_code == 200
+
+        spans = [s for _, _, s in interfaces.library.get_spans(request=request)]
+        assert spans, "No spans to validate"
+        for span in spans:
+            meta = span.get("meta", {})
+            validation(meta)
+
+    def setup_rc(self):
+        self.disabled = self._send_rc_and_execute_request(*DISABLED)
+        self.identification = self._send_rc_and_execute_request(*IDENTIFICATION)
+        self.anonymization = self._send_rc_and_execute_request(*ANONYMIZATION)
```

Comment:
This is why the CI is failing:
```
INTERNALERROR>   File "/home/runner/work/system-tests/system-tests/tests/appsec/test_automated_login_events.py", line 1792, in setup_rc
INTERNALERROR>     self.disabled = self._send_rc_and_execute_request(*DISABLED)
INTERNALERROR>                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
INTERNALERROR> TypeError: Test_V3_Login_Events_RC._send_rc_and_execute_request() takes 2 positional arguments but 3 were given
```

I don't know python enough to be sure about why is happening this, but I guess that we should move the `*` in `*DISABLED`, `*IDENTIFICATION` and `*ANONYMIZATION` to line 1775:
```python
        config_state = rc.rc_state.set_config(*config).apply()
```

```python
        self.disabled = self._send_rc_and_execute_request(DISABLED)
        self.identification = self._send_rc_and_execute_request(IDENTIFICATION)
        self.anonymization = self._send_rc_and_execute_request(ANONYMIZATION)
```